### PR TITLE
Use post for redeploy

### PR DESCRIPTION
**Pull request by k0nserv**
I am not sure about testing this locally. It seems like it's not possible to build and run cocoadocs locally in a simple way? Will this change require updates anywhere else or is this the only place?

cc @orta

### CODE REVIEW

File: views/404.slim [hunk 1]
```diff
@@ -132,23 +134,28 @@ javascript:
...
-
-      reparse.innerHTML = "Thanks, reparsing right now. Please give it some time and come back in 5-10 mins."
-      reparse.disabled = true
-    } else {
-
+    if (pathbits.length <= 2) {
       reparse.innerHTML = "Could not find the Podname from your address, please check a docs full URL."
     }
+
+    var address = "http://199.229.252.196:4567/redeploy/" + pathbits[2]
```
Comment (k0nserv) [inline views/404.slim]: Is there a DNS entry for this server? api.cocoadocs.org does not resolve to this IP oddly enough.

Comment (orta) [inline views/404.slim]: we should make it api.cocoadocs :+1: - it could be the old server address

**Comment by segiddins** [comment]
@orta @k0nserv what's the status on this?

**Comment by k0nserv** [comment]
Not sure. I think it's in @orta's corner. He was going to update the DNS i believe

**Comment by k0nserv** [comment]
@orta Status on this? Anything I can do?

**Comment by orta** [comment]
It's more that creating and updating the [README](http://cocoadocs.org/readme/) / 404 page / my blog isn't automated, and it's not high on my priorities yet

**Comment by k0nserv** [comment]
Okay, I'll keep this PR up to date until you find the time then :)

**Comment by orta** [comment]
OK, I think I'm going to not merge this, as the person who mostly uses the redeploy feature I don't want to force myself to have to jump through a bunch of hoops just to redeploy a pod, or force everyone else to. Otherwise it's going to sit as an idle PR forever.

**Comment by k0nserv** [comment]
Sounds reasonable :thumps_up:

**Pull request closed without merging**